### PR TITLE
API: Document /force-reload endpoint

### PR DESCRIPTION
**Pull request by fflorent**
Document the `/force-reload` endpoint, and therefore make it available in the Console API

### CODE REVIEW

File: api/grist.yml [hunk 1]
```diff
@@ -443,6 +443,18 @@ paths:
             text/json:
               schema:
                 $ref: '#/components/schemas/TableSchemaResult'
+  /docs/{docId}/force-reload:
+    post:
+      operationId: forceReload
+      tags:
+        - docs
+      summary: "Reload a document"
+      description: "Close and reopen the document, forcing the python engine to retart."
```
Comment (hexaltation) [inline api/grist.yml:452]: May be conjugate the description at the third person as a *it*
```suggestion
      description: "Closes and reopens the document, forcing the python engine to restart."
```

**Review by hexaltation — COMMENTED** [review]
Small typo in `description` fix and maybe a conjugation suggestion.

**Review by hexaltation — APPROVED** [review]
LGTM

File: api/grist.yml [hunk 2]
```diff
@@ -443,6 +443,18 @@ paths:
...
+      operationId: forceReload
+      tags:
+        - docs
+      summary: "Reload a document"
+      description: "Closes and reopens the document, forcing the python engine to restart."
+      parameters:
+        - $ref: '#/components/parameters/docIdPathParam'
+      responses:
+        200:
+          description: Document reopened successfully
```
Comment (vviers) [inline api/grist.yml:457]:
```suggestion
          description: Document reloaded successfully
```

Comment (fflorent) [reply api/grist.yml:457]: Actually the document is closed and reopened. However we may reuse the term `reload` if everyone is more comfortable with that term.

Comment (vviers) [reply api/grist.yml:457]: The main interest of this endpoint is to reload the sandbox imo and the description should reflect it

**Pull request merged**